Resources/config/config.xml

```diff
@@ -58,8 +58,8 @@
             <tag name="twig.extension" alias="fos_message" />
         </service>
 
-        <service id="fos_message.search_query_factory.default" class="FOS\MessageBundle\Search\QueryFactory" scope="request" public="true">
-            <argument type="service" id="request" />
+        <service id="fos_message.search_query_factory.default" class="FOS\MessageBundle\Search\QueryFactory" public="true">
+            <argument type="service" id="request_stack" />
```

Comment:
I don't, 2.3 is still supported when we should as well. Let's keep it this way and not throwing a depreciation yet.